src/helpers/validation.ts

```diff
@@ -25,3 +25,8 @@ export const followersSchema = yup.object({
   follower: yup.string().required("followerId can't be blank"),
   following: yup.string().required("ownerId can't be blank"),
 });
+
+export const likesQuerySchema = yup.object({
+  tweet: yup.string(),
```

Comment:
strange Schema. do you really sure that this schema needed ?

Reply:
yes, but it should be updated